src/components/Nav/NavTop.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FaRegHeart } from 'react-icons/fa';
+import { FaUserCircle } from 'react-icons/fa';
+import { FaChevronDown } from 'react-icons/fa';
+import { FaSearch } from 'react-icons/fa';
+
+function NavTop() {
+  return (
+    <Header>
+      <Nav>
+        <WrapInfo>
+          <Logo>IOI</Logo>
+          <Subscribe>구독</Subscribe>
+          <Store>스토어</Store>
+          <WrapSearch>
+            <Input />
```

Comment:
```suggestion
            <Input placeholder=""/>
```
스타일 컴포넌트에 attrs라고 주는 것 보다 위와 같은 방법으로 컴포넌트에 직접 적용하는 방식이 더 좋겠네요!